Client/src/Components/product-reviews/Body.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import css from "./styles.css";
+
+// Creates body portion of a Review. Displays body text and photos if there are any.
```

Comment:
Is there a reason why you are destructering props on line 7 and not with the () arguments list?
Your img element on line 15 should have meaningful alt text, maybe "user submitted image of product x".
There is a typo in the img property "height"